client/performance-profiler/hooks/use-support-chat-llm-query.ts

```diff
@@ -1,46 +1,40 @@
 import { useQuery } from '@tanstack/react-query';
+import { PerformanceMetricsItemQueryResponse } from 'calypso/data/site-profiler/types';
 import wp from 'calypso/lib/wp';
 
 function mapResult( response: WPComSupportQueryResponse ) {
 	return response.messages?.[ 0 ]?.content ?? '';
 }
 
 export const useSupportChatLLMQuery = (
-	title: string,
-	context: string,
+	insight: PerformanceMetricsItemQueryResponse,
 	hash: string,
 	is_wpcom: boolean,
 	enable: boolean
 ) => {
-	const question = `I need to fix the following issue to improve the performance of my WordPress site: ${ title }.`;
-	const howToAnswer =
-		'Answer me in two topics in bold: "Why is this important?" and "How to fix this?".';
-	const contextToInclude = `Use each of the following context: ${ context }. Visit any links provided for more information.`;
-	const suggestionsToInclude =
-		'Add if the issue can be fixed automatically with a WordPress setting, an editor block setting, or a plugin. Prefer Automattic owned plugins. Add relevant links. Provide code snippets as a last resort.';
-	const wpcomOrSelfHostedSuggestions = ! is_wpcom
-		? 'Do not show suggestions that work only in WordPress.com. Do not show any references to documentation or support forums related to WordPress.com.'
-		: 'Provide suggestions that work only in WordPress.com. Use WordPress.com documentation if availabble.';
-	const otherDirectives =
-		'Do not suggest doing additional tests to identify the issue. Do not suggest any other tool/website/service for doing another test except WordPress Speed Test (https://wordpress.com/speed-test). Do not mention Lighthouse reports or any other tool. Rarely suggest subscribing to weekly emails feature of the Performance Profiler tool to monitor changes. Do not ask follow-up questions or offer further assistance or ask for any feedback.';
-	const message = `${ question } ${ howToAnswer } ${ contextToInclude } ${ suggestionsToInclude } ${ wpcomOrSelfHostedSuggestions } ${ otherDirectives }`;
+	const question = `I need to fix the following issue to improve the performance of my WordPress site: ${ insight.title }.`;
```

Comment:
Considering the prompt is being created on the backend, why do we still need to even pass the question/message?

IMHO we should be able to pass only the insight object and the backend takes it from there. What do you think?

Reply:
Thanks for the suggestion @WBerredo. I have addressed that in [ad7bad0](https://github.com/Automattic/wp-calypso/pull/95580/commits/ad7bad0265afabf0dcf8f54f555076d6c7d64293).